### ISSUE TITLE
Leftovers did not discover or delete GCP image

### ISSUE THREAD

**Issue by pivotal-gabriel-dumitrescu** [genevieve/leftovers#61]
Image resource named  `ultrapink-ops-manager-image` was not discovered during --dry-run or deleted when ran with --no-confirm

`$ leftovers --filter ultrapink --iaas=gcp --gcp-service-account-key=service-account-key.json`

```
$ leftovers -v
v0.37.0
```

**Comment by genevieve** [comment]
That's really weird! It should be passing this constraint and adding it to list of resources to delete. https://github.com/genevieve/leftovers/blob/v0.37.0/gcp/compute/images.go#L38

Is it possible that the image is in a different gcp project?

**Comment by pivotal-gabriel-dumitrescu** [comment]
We used gcloud cli to list images and we got the following list: 
[gcloud-list.txt](https://github.com/genevieve/leftovers/files/2328712/gcloud-list.txt)

Next we modified leftovers to print out a list of images it found:
[leftovers-list.txt](https://github.com/genevieve/leftovers/files/2328711/leftovers-list.txt)

We discovered that leftovers cli stops listing the images when it finds stemcells in the list of images.

TLDR of files:
**GCLOUD CLI**
< kellygreen-ops-manager-image
< laurelgreen-ops-manager-image        
< macaroniandcheese-ops-manager-image  
< madderlake-ops-manager-image         
< majorelleblue-ops-manager-image      
< maroon-ops-manager-image             
< msugreen-ops-manager-image           
< mustardbrown-ops-manager-image       
< myrtlegreen-ops-manager-image
...

< LONG LIST OF STEMCELLS >

...
< stizza-ops-manager-image             
< stopred-ops-manager-image            
< stormcloud-ops-manager-image         
< strawberrydaiquiri-ops-manager-image 
< sunburntcyclops-ops-manager-image    
< sunray-ops-manager-image             
< superpink-ops-manager-image          
< taobaoorange-ops-manager-image       
< teagreen-ops-manager-image           
< tealgreen-ops-manager-image          
< tearose-ops-manager-image            
< telemagenta-ops-manager-image        
< trolleygrey-ops-manager-image        
< tropicalviolet-ops-manager-image     
< tulip-ops-manager-image              
< turkishrose-ops-manager-image        
< turquoiseblue-ops-manager-image      
< twilightlavender-ops-manager-image   
< usafablue-ops-manager-image          
< vanillaice-ops-manager-image         

---------
**LEFTOVERS CLI**
> kellygreen-ops-manager-image
> laurelgreen-ops-manager-image
> macaroniandcheese-ops-manager-image
> madderlake-ops-manager-image
> majorelleblue-ops-manager-image
> maroon-ops-manager-image
> msugreen-ops-manager-image
> mustardbrown-ops-manager-image
> myrtlegreen-ops-manager-image



**Modified code in leftovers to print the list:**
```golang
func (i Images) List(filter string) ([]common.Deletable, error) {
	images, err := i.client.ListImages()
	if err != nil {
		return nil, fmt.Errorf("List Images: %s", err)
	}

	var resources []common.Deletable
	for idx := range images.Items {
		img := &images.Items[idx].Name
		fmt.Println("IMAGE", *img)

		resource := NewImage(i.client, *img)
		if !strings.Contains(*img, filter) {
			continue
		}

		proceed := i.logger.PromptWithDetails(resource.Type(), resource.Name())
		if !proceed {
			continue
		}

		resources = append(resources, resource)
	}
	return resources, nil
}
```

**Comment by genevieve** [comment]
Okay so it seems like the `ImagesListCall` in the gcp compute api library returns a default of 500 images and there are way more than that in your project! :-)

I'll add looping over the pages to the client to grab the rest!

**Comment by genevieve** [comment]
This should do it: https://github.com/genevieve/leftovers/blob/8876507627e54e087e534fe18d77a7e1132de156/gcp/compute/client.go#L125-L146

Will cut a release this morning!

**Comment by pivotal-gabriel-dumitrescu** [comment]
Thank you for the quick turnaround.

**Comment by genevieve** [comment]
No worries! This should be in the latest release `v0.38.0`, lmk if there's any trouble.
